FlaskApp/templates/admin_pages/blog_new.html

```diff
@@ -1,4 +1,9 @@
-{% extends "base.html" %}
+{% extends "base.html" %}\
+{% block additional_tags %}
+<link href='https://cdn.jsdelivr.net/npm/froala-editor@3.0.0/css/froala_editor.pkgd.min.css' rel='stylesheet' type='text/css' />
+<script type='text/javascript' src='https://cdn.jsdelivr.net/npm/froala-editor@3.0.0/js/froala_editor.pkgd.min.js'></script>
+
```

Comment:
Let's clean this up

Remove the extra line in additional_tags
Put a linebreak above it